pytest_helm_charts/giantswarm_app_platform/utils.py

```diff
@@ -1,22 +1,33 @@
-from typing import List
+from typing import List, Optional
 
-from pykube import HTTPClient
+import pykube
+import yaml
+from pykube import HTTPClient, ConfigMap
 
 from pytest_helm_charts.giantswarm_app_platform.custom_resources import AppCR
-from pytest_helm_charts.utils import wait_for_namespaced_objects_condition
+from pytest_helm_charts.giantswarm_app_platform.entities import ConfiguredApp
+from pytest_helm_charts.utils import wait_for_namespaced_objects_condition, YamlDict
 
 
-def _app_deployed(app: AppCR) -> bool:
+def _app_has_status(app: AppCR, status: str) -> bool:
     complete = (
         "status" in app.obj
         and "release" in app.obj["status"]
         and "appVersion" in app.obj["status"]
         and "status" in app.obj["status"]["release"]
-        and app.obj["status"]["release"]["status"] == "deployed"
+        and app.obj["status"]["release"]["status"].lower() == status
```

Comment:
```suggestion
        and app.obj["status"]["release"]["status"] == status
```

Do we still need this? With helm 3 the statuses are always lower case.

Reply:
I would keep it; it's not a big performance impact and I remember this being true for Helm2. Maybe some people (outside GS) still use that?

Reply:
Yes Helm 2 uses upper case. OK sure that's fine to keep it.